spring-integration-zip/src/main/java/org/springframework/integration/zip/splitter/UnZipResultSplitter.java

```diff
@@ -32,21 +33,37 @@
  * @since 1.0
  *
  */
-public class UnZipResultSplitter {
-
-	public List<Message<Object>> splitUnzippedMap(Map<String, Object> unzippedEntries) {
+public final class UnZipResultSplitter {
```

Comment:
Not sure that making it as a `final` is a good idea.
Someone may decide to extend the logic but still reuse our solution.

At last it's up to end-user to extend the class on his own faith.
Since this class doesn't provide an access to any critical Framework block, that is really an overhead to protect it with `final`.